graphql-server/pkg/utils/graphql.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/vektah/gqlparser/v2/ast"
+)
+
+// get argument by name from root field (ie query or mutation) from resolver context
+func GetOperationArgument(ctx context.Context, name string) (*ast.Argument, error) {
```

Comment:
I assume we can drop this file if we move on to reading unauthed user from header, or enforce all user to auth at connect wallet